fauxfactory/__init__.py

```diff
@@ -634,8 +642,8 @@ def gen_mac(delimiter=':', multicast=None, locally=None):
 
     octets = [first_octet]
     octets.extend([
-        random.randint(0, 255) for _ in range(5)
-    ])
+                      random.randint(0, 255) for _ in range(5)
+                  ])
```

Comment:
Sometimes Pycharm reformatting sucks. But at least I put the code in one line only and without using list.